src/main/java/com/datatorrent/netlet/DefaultEventLoop.java

```diff
@@ -513,14 +512,18 @@ public String toString()
 
             });
           }
-        }
-        catch (IOException ie) {
-          l.handleException(ie, DefaultEventLoop.this);
+        } catch (UnresolvedAddressException e) {
+          logger.error("Inet Address {} is not resolvable.", address, e);
```

Comment:
error/warn/info - please do not do logging that that level. May be use trace or debug.